eox_core/api/data/v1/filters.py

```diff
@@ -10,6 +12,43 @@
 from eox_core.edxapp_wrapper.users import get_course_enrollment
 
 
+def check_python_version():
```

Comment:
I think that we should use six for this.

See: https://six.readthedocs.io/#six.PY2

Reply:
thanks but I changed the solution!

Following what @andrey-canon suggested:

Given the problem I described, we could update django-filters in Ironwood from 1.0.4 to 1.1.0, this particular version supports name (raises just a deprecation warning) and field_name [reference](https://github.com/carltongibson/django-filter/blob/1.1.0/django_filters/filters.py#L92) What do you think?